### PR TITLE
chore(4176): add auto approval usage metrics for last two weeks

### PR DESCRIPTION
**Pull request by Kolezhanchik**

### CODE REVIEW

File: app/services/k8s/usage-metrics.ts [hunk 1]
```diff
@@ -84,28 +127,32 @@ export async function getPodMetrics(
...
-            cpu: normalizeCpu(container.usage.cpu) || 0,
-            memory: normalizeMemory(container.usage.memory) || 0,
+            cpu: normalizeCpu(lastHourUsage.cpu) as number,
+            memory: normalizeMemory(lastHourUsage.memory) as number,
           },
           limits: {
-            cpu: normalizeCpu(resourceDef.limits?.cpu || '0'),
-            memory: normalizeMemory(resourceDef.limits?.memory || '0'),
+            cpu: normalizeCpu(resourceDef.limits?.cpu || 0) as number,
+            memory: normalizeMemory(resourceDef.limits?.memory || 0) as number,
```
Comment (junminahn) [inline app/services/k8s/usage-metrics.ts:147]: let's remove `as number`

File: app/services/k8s/usage-metrics.ts [hunk 2]
```diff
@@ -84,28 +127,32 @@ export async function getPodMetrics(
...
           },
           requests: {
-            cpu: normalizeCpu(resourceDef.requests?.cpu || '0'),
-            memory: normalizeMemory(resourceDef.requests?.memory || '0'),
+            cpu: normalizeCpu(resourceDef.requests?.cpu || 0) as number,
+            memory: normalizeMemory(resourceDef.requests?.memory || 0) as number,
```
Comment (junminahn) [inline app/services/k8s/usage-metrics.ts:151]: let's remove `as number`

File: app/services/k8s/usage-metrics.ts [hunk 3]
```diff
@@ -84,28 +127,32 @@ export async function getPodMetrics(
     // Map over containers to collect their usage, limits, and requests
     const containers = item.containers
       .filter((container) => {
-        if (container.name === 'POD' && container.usage.cpu === '0' && container.usage.memory === '0') return false;
+        if (container.name === 'POD') return false;
         return true;
       })
       .map((container, index) => {
         const resourceDef = podStatus.body.spec?.containers[index]?.resources ?? {};
```
Comment (junminahn) [inline app/services/k8s/usage-metrics.ts:134]: is it reliable to find the container by index, if the container has name, is it better to use the name to fine it? if then, is the name always unique?

Comment (Kolezhanchik) [reply app/services/k8s/usage-metrics.ts:134]: When querying pod metrics, Kubernetes adds a pseudo-container named `POD` to represent the pod's management overhead, like network or logging. This is not an actual application container and usually shows zero resource usage `(cpu: '0', memory: '0')`. looks like usually it append as the last array item, but not sure if it is a good idea to relay on `last index`

**Review by junminahn — APPROVED** [review]

**Pull request merged**